### PR TITLE
Added curl to wardend container

### PR DESCRIPTION
**Pull request by mikolajsobolewski**
## Summary by CodeRabbit

- **Chores**
  - Added `curl` to the Dockerfile for improved functionality and compatibility.

### CODE REVIEW

**Review by pitasi — CHANGES_REQUESTED** [review]
could you squash the two commits into a single one and rename it into something like "build: add curl to wardend container"? Many thanks!

**Comment by mikolajsobolewski** [comment]
it's not mine changes. It's only update this branch from `main`

File: Dockerfile [hunk 1]
```diff
@@ -26,7 +26,7 @@ RUN --mount=type=bind,source=.,target=.,readonly\
 
 FROM debian:bookworm-slim AS wardend
 RUN apt update && \
-    apt install ca-certificates -y && \
+    apt install ca-certificates curl -y && \
```
Comment (jlehtimaki) [inline Dockerfile:29]: I think we need `tar` ?

**Review by jlehtimaki — CHANGES_REQUESTED** [review]

File: Dockerfile (hunk 1 above)
Comment (mikolajsobolewski) [reply Dockerfile:29]: We don't need install the tar. Image already contains the tar.

**Review by pitasi — APPROVED** [review]

**Review by jlehtimaki — APPROVED** [review]

**Pull request merged**